index.html

```diff
@@ -1,9 +1,502 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html lang="en">
 <head>
     <meta charset="utf-8">
+    <link href="index.css" rel="stylesheet">
     <title>Задача «Мяндекс.Муррркет»</title>
 </head>
 <body>
+<input id="tile-layout-switch" class="tile-layout-switch" type="radio" name="switch" checked>
+<input id="card-layout-switch" class="card-layout-switch" type="radio" name="switch">
+<header class="main-header">
+    <h1>
+        Catdex.Market
+    </h1>
+    <div class="layout-switches">
+        <label for="card-layout-switch">Карточки</label>
+        <label for="tile-layout-switch">Плитки</label>
+    </div>
+</header>
+<main>
+    <article class="market-item">
+        <div class="item-picture">
+            <img src="pictures/longcatislong.jpg" alt="Cat longcat">
+        </div>
+        <section class="item-info">
```

Comment:
https://www.w3schools.com/html/html5_semantic_elements.asp прямо указывает на то, что article может содержать section и наоборот.
>Example for a newspaper: The sport articles in the sport section, may have a technical section in each article.

И здесь тоже: https://html.spec.whatwg.org/multipage/sections.html#article-or-section
>On the other hand, one could imagine a tweet or reddit comment or tumblr post or newspaper classified ad that just said "Granny Smith. These juicy, green apples make a great filling for apple pies."; it would then be articles because that was the whole thing.

>A section forms part of something else. 

Тем более, что article имеет ещё и значение "вещь/товар".

Reply:
Окей. Да, так можно. А зачем ты вообще обернул это в `section`? Сргласна с `article` выше. Почему именно `section`, а не простой `div`?